product-mini/platforms/common/wasm_proposal.c

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (C) 2023 Amazon.com Inc. or its affiliates. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+ */
+
+#include <stdio.h>
+
+void
+wasm_proposal_print_status(void)
+{
+    printf("Phase4 Wasm Proposals Status:\n");
+    printf("  Always-on:\n");
+    printf("    - Extended Constant Expressions\n");
+    printf("    - Multi-value\n");
+    printf("    - Non-trapping float-to-int conversions\n");
+    printf("    - Sign-extension operators\n");
+    printf("    - WebAssembly C and C++ API\n");
+    printf("  Configurable:\n");
+    printf("    - Bulk Memory Operation: %u\n", WASM_ENABLE_BULK_MEMORY);
```

Comment:
Bulk memory operations are technically no longer in Phase4; it's a finished proposal: https://github.com/WebAssembly/proposals/blob/main/finished-proposals.md